Makefile

```diff
@@ -1,20 +1,29 @@
 PREFIX?=/usr/local
 
+TEMPORARY_FOLDER=./tmp_portable_licenseplist
+
 build:
 	swift build -c release -Xswiftc -static-stdlib
- 
-test: 
+
+test:
 	swift test
 
 lint:
 	swiftlint
- 
+
 clean:
 	swift package clean
 
 xcode:
 	swift package generate-xcodeproj
-	
+
 install: build
 	mkdir -p "$(PREFIX)/bin"
 	cp -f ".build/release/LicensePlist" "$(PREFIX)/bin/license-plist"
+
+portable_zip: build
+	mkdir "$(TEMPORARY_FOLDER)"
+	cp -f ".build/release/LicensePlist" "$(TEMPORARY_FOLDER)"
+	cp -f "LICENSE" "$(TEMPORARY_FOLDER)"
+	(cd tmp/portable_licenseplist; zip -r - LICENSE LicensePlist) > './portable_licenseplist.zip'
```

Comment:
Is `tmp/portable_licenseplist` correct?
It seems to be that `$(TEMPORARY_FOLDER)`(`tmp_portable_licenseplist `) is correct for me.